packages/indexer-agent/src/agent.ts

```diff
@@ -839,6 +839,25 @@ class Agent {
       logger.info(`No active allocation for deployment, creating one now`, {
         allocationAmount: formatGRT(desiredAllocationAmount),
       })
+
+      // skip allocation if the previous allocation to the deployment closes with null or 0x00
```

Comment:
`// Skip allocating if the previous allocation for this deployment was closed with a null or 0x00 PoI`?